modules/kubernetes-addons/aws-load-balancer-controller/main.tf

```diff
@@ -1,81 +1,9 @@
- * OF CONTRACT, TORT OR OTHERWISE, ARISING FROM, OUT OF OR IN CONNECTION WITH THE
- * SOFTWARE OR THE USE OR OTHER DEALINGS IN THE SOFTWARE.
- */
-
-resource "helm_release" "lb_ingress" {
-  count                      = var.manage_via_gitops ? 0 : 1
-  name                       = local.helm_config["name"]
-  repository                 = local.helm_config["repository"]
-  chart                      = local.helm_config["chart"]
-  version                    = local.helm_config["version"]
-  namespace                  = local.helm_config["namespace"]
-  timeout                    = local.helm_config["timeout"]
-  values                     = local.helm_config["values"]
-  create_namespace           = local.helm_config["create_namespace"]
-  lint                       = local.helm_config["lint"]
-  description                = local.helm_config["description"]
-  repository_key_file        = local.helm_config["repository_key_file"]
-  repository_cert_file       = local.helm_config["repository_cert_file"]
-  repository_ca_file         = local.helm_config["repository_ca_file"]
-  repository_username        = local.helm_config["repository_username"]
-  repository_password        = local.helm_config["repository_password"]
-  verify                     = local.helm_config["verify"]
-  keyring                    = local.helm_config["keyring"]
-  disable_webhooks           = local.helm_config["disable_webhooks"]
-  reuse_values               = local.helm_config["reuse_values"]
-  reset_values               = local.helm_config["reset_values"]
-  force_update               = local.helm_config["force_update"]
-  recreate_pods              = local.helm_config["recreate_pods"]
-  cleanup_on_fail            = local.helm_config["cleanup_on_fail"]
-  max_history                = local.helm_config["max_history"]
-  atomic                     = local.helm_config["atomic"]
-  skip_crds                  = local.helm_config["skip_crds"]
-  render_subchart_notes      = local.helm_config["render_subchart_notes"]
-  disable_openapi_validation = local.helm_config["disable_openapi_validation"]
-  wait                       = local.helm_config["wait"]
-  wait_for_jobs              = local.helm_config["wait_for_jobs"]
-  dependency_update          = local.helm_config["dependency_update"]
-  replace                    = local.helm_config["replace"]
-
-  postrender {
-    binary_path = local.helm_config["postrender"]
-  }
-
-  dynamic "set" {
-    iterator = each_item
-    for_each = local.helm_config["set"] == null ? [] : local.helm_config["set"]
-
-    content {
-      name  = each_item.value.name
-      value = each_item.value.value
-    }
-  }
-
-  dynamic "set_sensitive" {
-    iterator = each_item
-    for_each = local.helm_config["set_sensitive"] == null ? [] : local.helm_config["set_sensitive"]
-
-    content {
-      name  = each_item.value.name
-      value = each_item.value.value
-    }
-  }
-
-  depends_on = [module.irsa_addon]
+module "helm_addon" {
```

Comment:
do we need `set_values` for `serviceAccount.name` and `serviceAccount.create` ?

Reply:
I found it in https://github.com/aws-samples/aws-eks-accelerator-for-terraform/blob/refactor/lb-controller-helm-addons/modules/kubernetes-addons/aws-load-balancer-controller/values.yaml#L4-L5 which is being used here https://github.com/aws-samples/aws-eks-accelerator-for-terraform/blob/refactor/lb-controller-helm-addons/modules/kubernetes-addons/aws-load-balancer-controller/locals.tf#L21